woocommerce/woocommerce
# Display update quantity notice on classic cart

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:





Closes #42747.

In https://github.com/woocommerce/woocommerce-blocks/pull/12043, we replaced the old notice styles with the new ones. Due to this change, the update quantity notice on the classic cart stopped working. The notices to display are fetched in https://github.com/woocommerce/woocommerce/blob/be290984b9e0d7228b2985022ceb7e7362094493/plugins/woocommerce/client/legacy/js/frontend/cart.js#L91

When introducing the new notice styles, we also introduced new CSS classes for the notices.

**Old class names:**

```css
.woocommerce-error
.woocommerce-message
.woocommerce-info
```

**New class names:**

```css
.is-error
.is-success
.is-notice
```



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a test page and add the following shortcode to it:
```
[woocommerce_cart]
```
2. Go to the frontend and add a product to the cart.
3. Go to the created test page with the cart shortcode.
4. Adjust the product quantity and click the `Update cart` button.
5. Verify that the notice `Cart updated.` is displayed.

<table>
<tr>
<td valign="top">Before:
<br><br>
<img width="1278" alt="Screenshot 2023-12-13 at 17 03 40" src="https://github.com/woocommerce/woocommerce/assets/3323310/04ce8f05-e9c5-4e7e-b6dd-e81e910e4e8d">
</td>
<td valign="top">After:
<br><br>
<img width="1279" alt="Screenshot 2023-12-13 at 17 02 54" src="https://github.com/woocommerce/woocommerce/assets/3323310/6fc9f335-921a-4afe-836b-7110c3808945">
</td>
</tr>
</table>



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Fix: Display update quantity notice when using the cart shortcode.

#### Comment 

</details>
